unittests/APITests/InterruptableConditionVariable.cpp

```diff
@@ -0,0 +1,154 @@
+#include <catch2/catch.hpp>
+#include <chrono>
+#include <csetjmp>
+#include <FEXCore/Utils/InterruptableConditionVariable.h>
+#include <thread>
+#include <signal.h>
+
+// Test that ensures the Reentrant mutex will timeout without signaling
+TEST_CASE("SimpleWait.InterruptableConditionVariable") {
```

Comment:
Sorry that I have to comment in this line yet again 😅 But why tag each `TEST_CASE` with `InterruptableConditionVariable` instead of including this subcategory in the `TEST_SUFFIX` parameter given to `catch_discover_tests`? It's already clear what entity is being tested from the filename, so this seems like a nuisance waiting for copy-paste typos to happen.

Reply:
Because I've never used catch2 so I'm extrapolating everything from gtest knowledge.

Reply:
Ah, right. Catch does require unique test names but only within each individual test executable. Since each test compiles as its own executable, it's fine to move the subcategory to `catch_discover_tests` too.

Then again, merging all tests into a single Catch executable (in some later PR) could be nice to have, which would require manually suffixing the subcategory for uniqueness after all . Taking that into consideration I don't have a strong opinion on this topic anymore, so let's proceed either way :)

Reply:
Resolved by using the TEST_SUFFIX in cmake.